src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2549,6 +2555,7 @@ class misc_commandscript : public CommandScript
     {
         std::string name;
         Player* player;
+        Unit* target = handler->getSelectedUnit();
```

Comment:
null check?

Reply:
got a check in line 2594, hopefully it is what you ment. If not then im a bit confused in how you would want the null check to be done exactly in this case.